src/main/java/org/interledger/codecs/oer/OerIA5StringCodec.java

```diff
@@ -41,15 +39,15 @@ public OerIA5String read(
   }
 
   @Override
-  public void write(
-      final CodecContext context, final OerIA5String instance, final OutputStream outputStream
-  ) throws IOException {
+  public void write(final CodecContext context, final OerIA5String instance,
+      final OutputStream outputStream) throws IOException {
 
     Objects.requireNonNull(context);
     Objects.requireNonNull(instance);
     Objects.requireNonNull(outputStream);
 
-    final byte[] data = instance.getValue().getBytes();
+    final byte[] data = instance.getValue()
+        .getBytes();
```

Comment:
it looks like your formatter has gone mad - surely this is not the best way to format this line (and many others too?)
We should probably look into why the formatters arent stable.